adam-core/src/main/scala/org/bdgenomics/adam/converters/TranscriptEffectConverter.scala

```diff
@@ -110,6 +147,7 @@ object VariantAnnotations extends Serializable with Logging {
 
     val te = TranscriptEffect.newBuilder()
     setIfNotEmpty(alternateAllele, te.setAlternateAllele(_))
+    // note: annotationImpact is not mapped
```

Comment:
The `annotationImpact` field (and variable above) is output by SnpEff version 4.2 but is not part of the VCF ANN specification, so I did not include it in our `TranscriptEffect` schema.